crates/graphql/src/schema.rs

```diff
@@ -919,11 +926,22 @@ impl QueryRoot {
             },
         );
 
+        let mint_addresses_subquery = token_accounts::table
+            .select(token_accounts::mint_address)
+            .filter(
+                token_accounts::owner_address
+                    .eq(any(owners.unwrap_or_else(Vec::new)))
+                    .and(token_accounts::amount.eq(1)),
+            );
+        let metadata_addresses_subquery = metadata_creators::table
+            .select(metadata_creators::metadata_address)
+            .filter(metadata_creators::creator_address.eq(any(creators.unwrap_or_else(Vec::new))));
+
         let rows: Vec<models::Nft> = query
             .filter(
-                metadatas::address.eq(any(metadata_creators::table
-                    .select(metadata_creators::metadata_address)
-                    .filter(metadata_creators::creator_address.eq(any(creators))))),
+                metadatas::address
+                    .eq(any(metadata_addresses_subquery))
+                    .or(metadatas::mint_address.eq(any(mint_addresses_subquery))),
```

Comment:
Another index for mint_address on metadatas table.

Reply:
this index is already added added